modules/core/shared/src/main/scala/io/circe/Json.scala

```diff
@@ -161,6 +161,26 @@ sealed abstract class Json extends Product with Serializable {
    * Use implementations provided by case classes.
    */
   override def hashCode(): Int
+
+
+  /**
+    * Recursively return all values matching the specified `key`.
+    *
+    * The Play docs, from which this method was inspired, reads:
+    *   "Lookup for fieldName in the current object and all descendants."
+    */
+  final def \\(key: String): List[Json] = keyValues(this).collect {
```

Comment:
Does `alias` mean that `findAllByKey` (or the other name) would have `public` access?